packages/object-reduce/README.md

```diff
@@ -0,0 +1,40 @@
+## just-reduce-object
+
+Part of a [library](../../../../) of zero-dependency npm modules that do just do one thing.
+Guilt-free utilities for every occasion.
+
+Like `Array.prototype.reduce` but for `Object`s.
+
+### Usage
+
+```js
+import reduce from 'just-reduce-object';
+
+reduce(
+  obj,
+  function predicate(previousValue, currentKey, currentValue, currentIndex, array) {},
+  initiaValue
+);
+```
+
+* `obj` – source object
+* `predicate` – function to execute on each key-value pair in the object
+* `initialValue` – optional initial value for the accumulator
+
+### Examples
+
+With `reduce`:
+```js
+reduce(obj, function (accumulator, key, value, idx, origin) {
+  accumulator[key] = value * 2;
+  return accumulator;
+}, {});
+```
+
+Without `reduce`:
+```js
+Object.keys(obj).reduce(function (accumulator, key, idx, origin) {
+  accumulator[key] = obj[key] * 2;
+  return accumulator;
+}, {});
+```
```

Comment:
Just copy main README and no need for "without reduce" example
 
```js
reduce({a: 3, b: 5, c: 9}, (acc, key, value) => { 
  acc[value]=key; 
  return acc; 
}, {}); // {3: 'a', 5: 'b', 9: 'c'}
```

Then show another example with an initial accumualtor